types/lib/stringifier.d.ts

```diff
@@ -1,14 +1,5 @@
-/**
- * @param {string} calc
- * @param {import('../parser').CalcNode} node
- * @param {string} originalValue
- * @param {{precision: number | false, warnWhenCannotResolve: boolean}} options
- * @param {import("postcss").Result} result
- * @param {import("postcss").ChildNode} item
- *
- * @returns {string}
- */
-export default function _default(calc: string, node: import('../parser').CalcNode, originalValue: string, options: {
+declare function _exports(calc: string, node: import('../parser').CalcNode, originalValue: string, options: {
     precision: number | false;
     warnWhenCannotResolve: boolean;
 }, result: import("postcss").Result, item: import("postcss").ChildNode): string;
+export = _exports;
```

Comment:
Does the TS export change make a noticeable difference for usage? This should be more correct as before TypeScript was checking ESM but we were really distributing CJS.